apps/expo/app/api/trpc/[trpc]+api.ts

```diff
@@ -0,0 +1,15 @@
+import { ExpoRequest, ExpoResponse } from "expo-router/server";
+import { fetchRequestHandler } from "@trpc/server/adapters/fetch";
+import { appRouter } from "@repo/trpc-server/routers/_app";
+
+export async function GET(req: ExpoRequest) {
+  const res = await fetchRequestHandler({
+    endpoint: "/api/trpc",
+    req: req as unknown as Request,
+    router: appRouter,
+    createContext: () => ({}),
+  });
+
+  // TODO: convert this to ExpoResponse.
```

Comment:
`ExpoRequest` and `ExpoResponse` are helper classes that have easy types and some statics like `.json()`. We augment the environment with Remix's Node.js polyfill for Request/Response so those can also be used. Overall the design leans into WinterCG and using the platform.